data-serving/data-service/src/model/case.ts

```diff
@@ -0,0 +1,43 @@
+import mongoose from 'mongoose';
+
+export const validOutcomes = ['pending', 'recovered', 'death'];
+
+const demographicsSchema = new mongoose.Schema({
```

Comment:
does mongoose have some sort of built-in way to parse a model object from a schema or do we have to roll our own validation? seems like a bummer to define the schema so nicely here but then still have to manually check each field in the request

Reply:
It might be tractable to simply remove document-specific validation from the controller, and to catch ValidationErrors from mongoose. We should include that in the above investigation.